.github/workflows/sync-next-branch-with-smithy-rs.yaml

```diff
@@ -0,0 +1,58 @@
+on: workflow_dispatch
+
+env:
+  rust_version: 1.54.0
+  java_version: 11
+
+name: Sync `next` branch with smithy-rs
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    name: Sync with smithy-rs
+    steps:
+      - name: Clone smithy-rs
+        uses: actions/checkout@v2
+        with:
+          repository: 'aws-labs/smithy-rs'
+      - uses: actions/cache@v2
+        name: Gradle Cache
+        with:
+          path: |
+            ~/.gradle/caches
+            ~/.gradle/wrapper
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+        # Pinned to the commit hash of v1.3.0
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: ${{ env.java_version }}
+      - name: Generate new SDK
+        working-directory: smithy-rs/
+        run: |
+          ./gradlew rust-runtime:assemble
+      - name: Clone aws-sdk-rust
+        uses: actions/checkout@v2
+        with:
+          repository: 'aws-labs/aws-sdk-rust'
+          path: 'next'
+      - name: Out with the old
+        working-directory: aws-sdk-rust
+        run: |
+          rm -rf sdk/*
+          rm -rf example/*
+      - name: In with the new
+        run: |
+          cp -a smithy-rs/rust-runtime/build/. aws-sdk-rust/
```

Comment:
This should probably delete the old contents first so that we don't have any orphaned files hanging out causing problems. Maybe `rm -rf aws-sdk-rust/sdk aws-sdk-rust/examples`.

Reply:
I do that in the preceding task 🙂

Reply:
Wow, I'm blind! 😅

Reply:
tunnel vision happens to the best of us